src/renderer/components/watch-video-playlist/watch-video-playlist.css

```diff
@@ -51,16 +51,22 @@
   border-radius: 50%;
   color: var(--tertiary-text-color);
   transition: background 0.2s ease-out;
+  stroke-width: 20;
+  stroke: var(--bg-color);
 }
 
 .playlistIcon:hover {
   background-color: var(--side-nav-hover-color);
   color: var(--side-nav-hover-text-color);
+  stroke-width: 10;
+  stroke: var(--side-nav-hover-text-color);
   transition: background 0.2s ease-in;
 }
```

Comment:
```suggestion
.playlistIcon:hover {
  background-color: var(--side-nav-hover-color);
  color: var(--side-nav-hover-text-color);
  transition: background 0.2s ease-in;
}
```

This looks great @sossost, thanks for doing this so quickly! My only suggestion is that we keep the inactive hover state the same, as otherwise this makes it seem like it's active.

Reply:
Thanks for the feedback, @jasonhenriquez! I appreciate your quick review. I agree with keeping the inactive hover state unchanged to avoid confusion about its activity status.
In this case, need to change the stroke color to the hover background color without deleting the style from the hover properties.
I used stroke color to make the icons thin without changing them.
<img width="42" alt="스크린샷 2024-04-14 오전 1 24 13" src="https://github.com/FreeTubeApp/FreeTube/assets/110542210/5b14e0be-46c1-455d-963a-8415116c4459">
<img width="46" alt="스크린샷 2024-04-14 오전 1 23 57" src="https://github.com/FreeTubeApp/FreeTube/assets/110542210/131bcb0c-4ca2-4357-864e-25482f435c35">


I'll make the necessary adjustments and update the pull request shortly.

Reply:
Thanks for the update! However, It's now not keeping the wider stoke width on active hover.

Reply:
Thanks for pointing that out! I missed updating the stroke width for the active hover state. I'll add a rule for `.playlistIconActive:hover` to ensure the wider stroke width and active color are maintained during hover. I'll make this change and update the pull request shortly.

Reply:
Thanks @sossost! This is working perfectly now. My last thought is that while we're here, can you make the `.playlistIconActive:hover`'s `stroke` & `color` be `--accent-color-hover`? I think this was another pre-existing bug I didn't even notice.

Reply:
Thanks, @ jasonhenriquez! I'm glad to hear it's working well now. I agree with your suggestion to update the `.playlistIconActive:hover` stroke and color to `--accent-color-hover`. It sounds like a good catch on a pre-existing issue. I'll make these changes and push the update soon.